packages/wrangler/src/type-generation/index.ts

```diff
@@ -207,15 +211,29 @@ export function generateImportSpecifier(from: string, to: string) {
  */
 export function constructType(
 	key: string,
-	value: string | number | boolean,
+	value: string | number | boolean | string[],
 	useRawVal = true
```

Comment:
Is this called with `false` anymore?

Reply:
Thanks so much for the comment! 🫶 

It turns out that things were not ok and I was breaking some functionality, this wasn't getting detected because we do have unit tests for the `constructType` function but we don't the same coverage in the integration tests

I've improved things and made sure that nothing is broken I also removed this function (which was, I believe complicating things) and converted the unit tests into integration ones

https://github.com/cloudflare/workers-sdk/pull/5086/commits/5de01e302d78a16acf2642b0d04406a5ffc7fb45